### PR TITLE
Fullcalendar button

### LINKED ISSUE #131: Add button on left sidebar to go to full calendar
image

(yellow button)

it should take user to `fullCalendar/index.html`

### PR DESCRIPTION
**Pull request by OasisAlex**
## Tracking Info

Resolves #131 and #129 

## Changes

- add the side bar full calendar button
- server route to the full calendar html
- button functionality passed with current year and month
- Fix bug for clicking date cell to main page but date not align.

## Testing

- test locally

## Confirmation of Change

image

### CODE REVIEW

File: source/sideCalendar/sideCalendar.js [hunk 1]
```diff
@@ -87,6 +87,8 @@ class SideCalendar extends HTMLElement {
                 display: flex;
                 flex-direction: column;
                 align-items: center;
+
+                gap: 5px;
```
Comment (EdwardLinS) [inline source/sideCalendar/sideCalendar.js:91]: Spaces out the buttons a little since they were touching

File: source/sideCalendar/Calendar.png
Comment (EdwardLinS) [inline source/sideCalendar/Calendar.png:1]: Is it possible to switch this out for an SVG? They're scalable and a lot better in resolution.

**Review by EdwardLinS — COMMENTED** [review]
Look great! The button matches the design in the Figma and also connects to the full calendar.

Good work Alex! Just some small requests, is it possible to change the png of the calendar icon to SVG?

File: source/sideCalendar/Calendar.png
Comment (OasisAlex) [reply source/sideCalendar/Calendar.png:1]: let me see what I can do about that

**Review by EdwardLinS — APPROVED** [review]
Nice, it looks good now! The last/next week buttons disappear in mobile view and match the Figma.

One small tweak, I changed the SVG for last/next week buttons because it was loading the wrong SVG before. Everything is perfect though, nice work! Thanks for making the changes.

**Pull request merged**